ir/memory.cpp

```diff
@@ -1039,6 +1066,16 @@ Memory::Memory(State &state) : state(&state), escaped_local_blks(*this) {
   if (has_null_block)
     alloc(expr::mkUInt(0, bits_size_t), bits_byte / 8, GLOBAL, false, false, 0);
 
+  if (has_fncall) {
+    // Fix the nonlocal block for encoding fn calls' side effects to be always
+    // alive & have GLOBAL type.
+    // Note that liveness of the block is already encoded by
+    // mk_nonlocal_liveness_array().
+    non_local_blk_kind.add(
```

Comment:
is this bit any useful?
It will just make the formula for the non-simplified case bigger. The liveness of this block is kinda irrelevant AFAICT.

Reply:
I think it is useful because this makes liveness bitvector after fn call simpler by keeping the live bit of this block.
It makes its interpretation consistent with `always_alive(unsigned)` as well.

Reply:
But it has the side-effect of increasing the size of the expression returned by `Pointer::getAllocType()`.
You can change the loop in `Memory::mkCallState()` instead to avoid the fncall bid.

Reply:
Ok, I take back. The current encoding uses UFs for non-local, so there's no formula size increase. Though I still prefer patchintg mkCallState. It's not worth to add this exception to the map.